packages/app/src/cli/utilities/app/http-reverse-proxy.ts

```diff
@@ -67,14 +53,38 @@ export async function runConcurrentHTTPProcessesAndPathForwardTraffic(
 
   const availablePort = portNumber ?? (await port.getRandomPort())
 
+  const proxy = httpProxy.createProxy()
+  const server = http.createServer(function (req, res) {
+    const target = match(rules, req)
+    if (target) return proxy.web(req, res, {target})
+
+    res.statusCode = 500
```

Comment:
I think we can provide a better experience here in this case -- `debug` statements for verbose mode probably useful.

Reply:
I can improve this, but it's impossible to reach this point because we are capturing every possible path (anything from `/`)